ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main.yaml

```diff
@@ -15,6 +31,12 @@ tests:
   commands: shellcheck ./clean-ci-resources.sh
   container:
     from: src
+- as: cleanup-vexxhost
+  cron: '@hourly'
```

Comment:
I'm not entirely sure how pj-rehearse deals with a cron job. The test seems to fail with no output, though. Do you have a clean execution of it already?

Reply:
nope, I forgot to set the `/hold` flag. Not a big deal though, and thanks for the LGTM